{"name": "Система вернёт ошибку, если в данных запроса указать неправильно логин или пароль", "status": "passed", "parameters": [{"name": "response_expected", "value": "{'code': 404, 'message': 'Учетная запись не найдена'}"}, {"name": "data", "value": "{'password': '0000000ef00000'}"}], "start": 1758314308621, "stop": 1758314308712, "uuid": "5158b490-4563-41c6-b1f9-f35be9059e92", "historyId": "0af060754728064e4619329806a2e403", "testCaseId": "375d36756d17fe61729a4bc9d5518957", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_creation_scenario_with_invalid_logins", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-U7O3CJE"}, {"name": "thread", "value": "7372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}], "titlePath": ["tests", "test_courier_login.py", "TestCourierLogin"]}